Sources/Knit/Container.swift

```diff
@@ -11,24 +11,11 @@ import Swinject
 
  The Knit.Container also performs the function of a weak wrapper of the `Swinject.Container`.
  */
-public class Container<TargetResolver>: Knit.Resolver {
+public class Container<TargetResolver: Resolver> {
 
     // MARK: - Knit.Resolver
 
-    public var resolver: TargetResolver {
-        self as! TargetResolver
-    }
-
-    /// Returns `true` if the backing container is still available in memory, otherwise `false`.
-    public var isAvailable: Bool {
-        _swinjectContainer != nil
-    }
-
-    // MARK: - Swinject.Resolver
-
-    public func unsafeResolver(file: StaticString, function: StaticString, line: UInt) -> Swinject.Resolver {
-        _unwrappedSwinjectContainer(file: file, function: function, line: line)
-    }
+    public let resolver: TargetResolver
```

Comment:
Does keeping this property access mean all the existing Container.register methods remain the same?